extensions/dicom-microscopy/src/DicomMicroscopyViewport.tsx

```diff
@@ -164,7 +164,11 @@ class DicomMicroscopyViewport extends Component {
         const image = new metadataUtils.VLWholeSlideMicroscopyImage({
           metadata: inst,
         });
-        const imageFlavor = image.ImageType[2];
+        const imageType =
```

Comment:
depending on different data source, `image.ImageType` sometimes comes in as a string, not a string array.